components/com_users/controllers/remind.php

```diff
@@ -46,14 +46,11 @@ public function remind()
 
 			return false;
 		}
-		else
-		{
-			// The request succeeded.
-			// Proceed to step two.
-			$message = JText::_('COM_USERS_REMIND_REQUEST_SUCCESS');
-			$this->setRedirect(JRoute::_('index.php?option=com_users&view=login', false), $message);
 
-			return true;
-		}
+		// To not expose if the user exists or not we send a generic message.
+		$message = JText::_('COM_USERS_REMIND_REQUEST');
+		$this->setRedirect(JRoute::_('index.php?option=com_users&view=login', false), $message);
```

Comment:
```suggestion
		$this->setRedirect(JRoute::_('index.php?option=com_users&view=login', false), $message, 'notice');
```

Display as a blue notice not a green message for consistency with reset request